pages/pipelines/build_exports.md

```diff
@@ -3,13 +3,13 @@
 > 📘 Enterprise feature
 > Build exports is only available on an [Enterprise](https://buildkite.com/pricing) plan, which has a [build retention](/docs/pipelines/build-retention) period of 12 months.
 
-If you need to retain build data beyond the [retention period](/docs/pipelines/build-retention) in your [Buildkite plan](https://buildkite.com/pricing), you can export the data to your own [Amazon S3 bucket](https://aws.amazon.com/s3/).
+If you need to retain build data beyond the [retention period](/docs/pipelines/build-retention) in your [Buildkite plan](https://buildkite.com/pricing), you can export the data to your own cloud object storage. We currently support [Amazon S3 bucket](https://aws.amazon.com/s3/) and [Google Cloud Storage bucket](https://cloud.google.com/storage/).
```

Comment:
@mitchbne exports was edited in its own wee PR :D
https://github.com/buildkite/docs/pull/2201/files

We can attempt to combine the two